obj.js

```diff
@@ -126,31 +115,39 @@ export default class BaseProtocol {
    * @param objDescrJSON {JSON} Describes the remote object, see PROTOCOL.md
    * @returns {StubObj}
    */
-  makeStub(objDescrJSON) {
+  async makeStub(objDescrJSON) {
     let proto = this._remoteClasses.get(objDescrJSON.className);
-    assert(proto, `Remote class ${ objDescrJSON.className } is unknown here. Make sure to first push the class description.`);
+    if (!proto) {
+      let classDescrJSON = await this.callRemote("class", {
+        idRemote: objDescrJSON.idLocal,
```

Comment:
I think the parameter should be the class name, no?

Reply:
The server doesn't have a mapping from class names back to prototypes that it would need to be able to do this, and I was worried that we would run into the issue of whether the server thinks the client knows about the class or not, and so I decided not to go that route.

Reply:
Can you please fix that in a new PR?